### DISCUSSION TITLE
git username related

### DISCUSSION THREAD

**Discussion by prati231999** [community/community#50512] [Actions]
### Select Topic Area

Bug

### Body

Following username skaterboy37 was updating my branch without acknowledge.. Or can say that whenever I tried to push code it came up as skaterboy37 which caused great effect to my work
Could you please help me in finding who this user is to prevent it from happening again

**Comment by airtower-luna** [comment]
Do I understand correctly that commits you made show up as made by another user? If yes, that means your user email setting is wrong.

Each time you make a commit, Git records the configured email address as the *author email* in the commit. GitHub uses the author email to detect which user made a commit. See [Setting your commit email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) and make sure to set an address connected to your account.

**Comment by prati231999** [comment]
Yes that's correct.. it worked!!
Then is the skaterboy37 author email?
It also had a profile picture with username

**Comment by airtower-luna** [comment]
> Then is the skaterboy37 author email?

That user must've added the email address that was set before you fixed it to your account. I don't know what that address was, but that's how GitHub matches commits with accounts.
